src/ngraph/runtime/backend.hpp

```diff
@@ -109,4 +109,9 @@ class ngraph::runtime::Backend
     virtual bool is_supported_property(const Property prop) const;
 
     virtual void remove_compiled_function(std::shared_ptr<Executable> exec);
+
+    // \brief Return a requested op from the backend if possible
+    // \param op_name is the name of the backend specific op
+    // \returns a shared pointer to the op if found, else nullptr
+    virtual std::shared_ptr<ngraph::op::Op> get_backend_op(const std::string& op_name, ...);
```

Comment:
I don't like the `...` here. Why not pass a second string which is the config string, or encode the config in the op_name and call it something else. If not then you really really need to put a description in the comment block on what this is. How do I use it? From the current comment I have no ideal now to implement `get_backend_op`.

Reply:
@rkimballn1 Adding more comments sounds like a great idea.

Reply:
Added more comments

Reply:
Since this is so backend specific, why not put it only in the backend in question. We don't need a generic Backend definition for a specific backend function. If you were to cast the backend you get back from Backend::create("CPU") into a CPUBackend class then you could directly access any custom op constructors without the need for variadic args